content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -46,6 +46,32 @@ further sub-isolations applied.
 In terms of Docker concepts, a Pod is similar to a group of Docker containers
 with shared namespaces and shared filesystem volumes.
 
+## Creating a Pod
+
+The following is an example of a Pod which consists of a container running the image `nginx:1.14.2`.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.14.2
+    ports:
+    - containerPort: 80
+```
+
+To create the Pod shown above, run the following command:
+```shell
+kubectl apply -f https://k8s.io/examples/pods/simple-pod.yaml
+```
+
```

Comment:
Do we want to add steps to list Pods and describe what the user sees as we have in the [deployments documentation](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment)?